db/migrate/20180312163100-create-score.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('scores', {
+      uri: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.STRING
+      },
+      unaware: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1,
+        validate: {
```

Comment:
`validate` should should only be on the model definition (models/db/*.model.js) and not the migration. It probably wont error here, but I don't think it adds any sort of database constraint.